README.ipynb

```diff
@@ -57,19 +57,19 @@
    "source": [
     "## Authorization\n",
     "\n",
-    "The Rescale API authorizes requests using an *API Key*. This key is unique per User, per Workspace. A give user may have multiple *API Keys* active - one per Workspace.\n",
+    "The Rescale API authorizes requests using an *API Key*. This key is unique per User, per Workspace. A given user may have multiple *API Keys* active.\n",
     "\n",
     "To generate an *API Key*, go to the API section of the User Profile.\n",
     "\n",
     "![](README.images/user_profile_apikey.png)\n",
     "\n",
     "> NOTE: Remember that your *API Key* is a secret. Never store it in a code repository. Never share it with your colleagues.\n",
     "\n",
-    "Rescale tools like Rescale CLI look for `apiconfig` authorization configuration file in users' home directory.\n",
+    "Rescale tools like Rescale CLI look for the `apiconfig` authorization configuration file in the users' home directory.\n",
     "\n",
     "```\n",
-    "$HOME/.config/rescale/apiconfig             # Linux\n",
-    "%USERPROFILE%\\config\\rescale\\apiconfig      # Windows\n",
+    "$HOME/.config/rescale/apiconfig             # Linux, macOS\n",
+    "%USERPROFILE%\\.config\\rescale\\apiconfig     # Windows\n",
```

Comment:
I tested this on Windows with the Rescale CLI. The correct path for the Rescale CLI has ".config", not "config".